scripts/run_queries_timescaledb.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Ensure runner is available
+EXE_FILE_NAME=${EXE_FILE_NAME:-$(which tsbs_run_queries_timescaledb)}
+if [[ -z "$EXE_FILE_NAME" ]]; then
+    echo "tsbs_run_queries_timescaledb not available. It is not specified explicitly and not found in \$PATH"
+    exit 1
+fi
+
+# Queries folder
+BULK_DATA_DIR=${BULK_DATA_DIR:-"/tmp/bulk_queries"}
+
+MAX_QUERIES=${MAX_QUERIES:-"0"}
+NUM_WORKERS=${NUM_WORKERS:-$(grep -c ^processor /proc/cpuinfo)}  # match # of cores - worker per core
```

Comment:
This does not work on non-Linux systems (e.g. MacOS or Windows), perhaps have this fall back to a reasonable default (4? 8?) if it fails?

Reply:
Yes, I agree that reasonable default would be useful in this case. PR updated accordingly with fallback to 4 workers in case `NUM_WORKERS` is not explicitly specified

Reply:
I do not see the change?

Reply:
It below, marked line, the whole code snippet looks like this:
```bash
NUM_WORKERS=${NUM_WORKERS:-$(grep -c ^processor /proc/cpuinfo)}  # match # of cores - worker per core
if [ -z "$NUM_WORKERS" ]; then
    # Workers number is not specified. Is /proc/cpuinfo available? Fallback to reasonable default
    NUM_WORKERS=4
fi
```

Reply:
Here is link directly [into code](https://github.com/sunsingerus/tsbs/blob/fb051cbfa5dfd01a437e21631cd039a0e382810f/scripts/run_queries_timescaledb.sh#L14)

Reply:
Yes, I agree this one-liner is more compact. Nice to learn :+1: 
Code updated